Sources/CodeEditTextView/TextView/TextView.swift

```diff
@@ -76,16 +76,22 @@ public class TextView: NSView, NSTextContent {
     }
 
     /// The default font of the text view.
+    /// - Note: Setting the font for the text view will update the font as the user types. To change the font for the
```

Comment:
Does this mean that only the text typed after changing the font will be updated or that the entire text will be updated after the user types?

Reply:
Only text typed after changing the font. Actually this API should probably be removed/renamed to make sure the difference is even more clear.